scalableminds/webknossos
# Track wk version to analytics

### URL of deployed dev instance (used for testing):
- https://gatrackversion.webknossos.xyz

### Steps to test:
- visit a few pages on the dev instance
- check network requests and check out GA report

### Issues:
- fixes #3763 

------
